halpybot/packages/utils/decorators.py

```diff
@@ -0,0 +1,187 @@
+"""
+decorators.py - All of the various utility decorators in use for the Seals
+
+Copyright (c) The Hull Seals,
+All rights reserved.
+
+Licensed under the GNU General Public License
+See license.md
+"""
+import functools
+from typing import List
+from loguru import logger
+from halpybot.packages.case import get_case
+from halpybot.packages.command import get_help_text
+from halpybot.packages.models import Case
+from ..exceptions import (
+    EDSMReturnError,
+    NoResultsEDSM,
+    EDSMLookupError,
+    EDSMConnectionError,
+)
+from .utils import cache_prep, sys_cleaner
+
+
+class CommandUtils:
```

Comment:
class used purely as a namespace :unamused:

Reply:
Already fixed in the next PR :D